drivers/flash/flash_mcux_flexspi_nor.c

```diff
@@ -19,6 +19,10 @@
 #define NOR_WRITE_SIZE	1
 #define NOR_ERASE_VALUE	0xff
 
+#ifdef CONFIG_FLASH_MCUX_FLEXSPI_NOR_WRITE_BUFFER
+static uint8_t flash_flexspi_nor_write_buffer[SPI_NOR_PAGE_SIZE];
```

Comment:
Would not shorter local name be clear enough here?
`static uint8_t nor_write_buf[SPI_NOR_PAGE_SIZE];`